tests/checks/performance/simple_model_comparison_test.py

```diff
@@ -197,5 +197,24 @@ def test_regression_tree_max_depth(diabetes_split_dataset_and_model):
     check = SimpleModelComparison(simple_model_type='tree', max_depth=5)
     # Act X
     result = check.run(train_ds, test_ds, clf).value
-    # Assert - 1 metrics X 2 models
-    assert_that(result['scores'], has_length(2))
+    # Assert
+    assert_regression(result)
+
+
+def assert_regression(result):
+    metric = DEFAULT_SINGLE_SCORER[ModelType.REGRESSION]
+    assert_that(result['scores'], has_entry(metric, has_entries({
+        'Origin': close_to(-100, 100), 'Simple': close_to(-100, 100)
+    })))
+    assert_that(result['scorers_perfect'], has_entry(metric, is_(0)))
+    assert_that(result['classes'], is_(None))
+
+
+def assert_classification(result, metrics=None):
+    metrics = metrics or [DEFAULT_SINGLE_SCORER_MULTICLASS_NON_AVG]
+    class_matchers = {clas: has_entries({'Origin': close_to(1, 1), 'Simple': close_to(1, 1)})
+                      for clas in result['classes']}
+    matchers = {metric: has_entries(class_matchers) for metric in metrics}
+    assert_that(result['scores'], has_entries(matchers))
+    assert_that(result['scorers_perfect'], has_entries({metric: is_(1) for metric in metrics}))
+    assert_that(result['classes'], contains_exactly(0, 1, 2))
```

Comment:
Don't want to thread on the toes of this [issue](https://github.com/deepchecks/deepchecks/issues/491), but may want to add a test for string labels...